t/e2e/transactions/transactions_test.go

```diff
@@ -83,14 +47,6 @@ func (s *TransactionsTestSuite) TestCallRPCSendTransaction() {
 		}]
 	}`)
 	s.NotContains(result, "error")
```

Comment:
Before we were comparing the Tx hash returned from SendTransaction to the actual result. Now that the actual result is the only thing returned, what should we compare to that result? Right now I'm just checking for no error. IN the RPC tests I'm not checking anything else. In the tests that call sendTransaction directly, I'm only checking that the response is not null.